tools/editor/editor_import_export.cpp

```diff
@@ -1441,7 +1441,7 @@ bool EditorExportPlatformPC::can_export(String *r_error) const {
 	String err;
 	bool valid=true;
 
-	if (use64 && (!exists_export_template(debug_binary64)) || !exists_export_template(release_binary64)) {
+	if ((use64 && (!exists_export_template(debug_binary64))) || !exists_export_template(release_binary64)) {
```

Comment:
Hmm true, I fixed it like that because it's the current interpretation by GCC (the warning tells you where to put the parentheses to remove the ambiguity), but what you're proposing makes more sense. Another (possible) bug uncovered then?